app/build.gradle.kts

```diff
@@ -17,7 +17,7 @@ android {
     compileSdk = 35
 
     defaultConfig {
-        applicationId = "rs.ruffle"
+        applicationId = "rs.ruffle.Ruffle"
```

Comment:
I'm googling around and it seems the examples should be lowercase, e.g. `com.example.appname` instead of `com.example.AppName` - could you find examples or documentation to the contrary?

Reply:
Hmm, good point. I could find _a few_ capitalized examples, but also a bunch where it's just two segments, like it is now:
![Screenshot_20250617-183007](https://github.com/user-attachments/assets/9107aac1-21e5-4e5b-b90a-54756be5778e)
![Screenshot_20250617-183119](https://github.com/user-attachments/assets/c36dd35b-aa56-4355-8cd3-5aa04cbf3361)
![Screenshot_20250617-183132](https://github.com/user-attachments/assets/b6f6af17-6425-4428-90b0-75335550bb52)
![Screenshot_20250617-183440](https://github.com/user-attachments/assets/f3fd9a9f-5ee9-4a9d-99bb-c3719caa8ce9)
![Screenshot_20250617-182939](https://github.com/user-attachments/assets/886e613f-ef87-497c-b236-74eceb399dc4)